app/actions/sessions.js

```diff
@@ -72,15 +73,21 @@ export function previewTables (tableNames) {
     return (_, getState) => {
         const state = getState();
         const sessionSelected = state.sessions.get('sessionSelected');
-        ipcSend(TASKS.PREVIEW, sessionSelected, tableNames);
+        const database = state.sessions.getIn(
+            ['list', sessionSelected, 'configuration']
+        ).toJS().database;
```

Comment:
could this just be `getIn(['list', sessionselected, 'configuration', 'database'])`?